sdk/include/cheri.hh

```diff
@@ -812,6 +812,20 @@ namespace CHERI
 			return permission_set_from_pointer(ptr);
 		}
 
+		/**
+		 * Drop some permissions from this capability.
+		 *
+		 * Because this function computes the permission mask by clearing bits
+		 * in the PermissionSet::omnipotent() all-ones mask, rather than from
+		 * the set of permissions currently held by this Capability, it is safe
+		 * to use to clear Global permission on a sealed capability.
+		 */
+		template<std::same_as<Permission>... Permissions>
+		void drop_permissions(Permissions... drop)
```

Comment:
We already have a without method on PermissionSet, please can you make the naming of this consistent?